setup/account_payment_select_cost_account/setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
```

Comment:
Hi @eLBati this is not a review but my curiousity
* What is setup tools is about? And normally this will be created automatically by ocabot, or we have to do it ourselves?
* If ocabot is doing it, should it present here in the PR?

Reply:
Hi @kittiu,
more info about setuptools-odoo: https://pypi.org/project/setuptools-odoo

This is normally created by ocabot, so you should not worry about adding it.

I explicitly add it when I need to deploy the module before having it released on pypi, using a `requirements.txt` line like this:
```
-e git+https://github.com/eLBati/account-payment.git@d1fc63d62970091a874fcaacaf3f0aadf805b2a9#egg=odoo12-addon-account-payment-select-cost-account&subdirectory=setup/account_payment_select_cost_account
```